app/controllers/application_controller.rb

```diff
@@ -7,10 +7,11 @@ class ApplicationController < ActionController::Base
   protected
 
   def devise_configure_permitted_parameters
-    devise_parameter_sanitizer.permit :invite, keys: [:email, :role, :registers]
+    devise_parameter_sanitizer.permit :invite, keys: [:email, :team_members]
```

Comment:
Does this work? I'd normally expect to see an array but I could be wrong `team_members: []`